server/views/pages/recommendations/searchPpud.njk

```diff
@@ -23,31 +23,31 @@
                         Name
                     </dt>
                     <dd class="govuk-summary-list__value">
-                          {{ recommendation.personOnProbation.fullName }}
+                    {{ hyphenForBlank(recommendation.personOnProbation.fullName) }}
                     </dd>
                 </div>
                 <div class="govuk-summary-list__row">
                     <dt class="govuk-summary-list__key">
                         Date of birth
                     </dt>
                     <dd class="govuk-summary-list__value">
-                        {{ formatDateTimeFromIsoString({isoDate: recommendation.personOnProbation.dateOfBirth}) }}
+                    {{ hyphenForBlank(recommendation.personOnProbation.dateOfBirth) }}
```

Comment:
Does this still format the date properly?

Reply:
Thanks Darren, nicely spotted!